mu4/notation/internal/notationinteraction.h

```diff
@@ -135,6 +136,14 @@ class NotationInteraction : public INotationInteraction
     void cmdAddSlur(const Ms::Slur* slurTemplate = nullptr);
     void addSlur(Ms::ChordRest* cr1, Ms::ChordRest* cr2, const Ms::Slur* slurTemplate);
 
+    struct HitMeasureData
```

Comment:
Let's avoid this struct, it may force others to create another structs like "HitSegmentData", "HitSystemData", etc.
All you need here is to add "staffIndex" field to Ms::Measure (maybe it already has one)

Reply:
decided to leave it as it is